### DISCUSSION TITLE
cannot run the bat after downloading the package

### DISCUSSION THREAD

**Discussion by michaelyeung445** [lllyasviel/Fooocus#3086] [Q&A] [answered]
it appear this 

OSError: [WinError 126] 找不到指定的模組。 Error loading "C:\Users\Micha\Downloads\Fooocus_win64_2-1-831\python_embeded\lib\site-packages\torch\lib\c10.dll" or one of its dependencies.

**Comment by Rogala** [comment, marked as answer]
[Microsoft Visual C++ Redistributable latest supported downloads](https://aka.ms/vs/17/release/vc_redist.x64.exe)

**Comment by SreeisonGithub** [comment]
@lllyasviel i have 4 gb  GTX1650 with i5 10 th gen processor, while trying to fooocus on my lappy, the process gets terminated while model moving to gpu. Is there any way i can solve this... i am noobie at this so pls help

**Reply by Rogala** [reply]
RAM of more than 16 GB and a swap file of 40 GB, preferably an SSD or NVMe drive

**Reply by mashb1t** [reply]
@SreeisonGithub please find the troubleshooting guide and valuable tips at https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md
